src/notebook_unit1.ipynb

```diff
@@ -706,7 +689,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# discrete Fourier transform of the remaining column: You only need the real frequencies"
+    "# discrete Fourier transform of the remaining column: You only need the real frequencies\n",
+    "data_f = np.fft.fft(data['y'])\n",
```

Comment:
This is exactly what I would have done. Inga does different stuff, calculating the frequency etc. I'd say we leave it like this for now and if we want to implement the functionality in our wonderful model we will just copy Ingas approach.